### ISSUE TITLE
Is there any idea to replicate the result using the keras code?

### ISSUE THREAD

**Issue by 14H034160212** [nuric/deeplogic#17]
Hi,

Do you have an issue with the replication to the same result? I tried to run the same code many times, but I got a different result each time.

**Comment by 14H034160212** [comment]
> Hi,
> 
> Do you have an issue with the replication to the same result? I tried to run the same code many times, but I got a different result each time.

**Comment by 14H034160212** [comment]
I try to add the following code at the top of the train.py and I run two programs at the same time. During the training process, the loss and acc are changing at the same page, but after the whole epoch, and do the validation. The two programs are not print the same number for the val_loss and val_acc. I do not why.
```
  os.environ["CUDA_VISIBLE_DEVICES"] = "6"
  seed_value = 0
  os.environ['PYTHONHASHSEED'] = str(seed_value)
  random.seed(seed_value)
  np.random.seed(seed_value)
  tf.set_random_seed(seed_value)

  session_conf = tf.ConfigProto(intra_op_parallelism_threads=1, inter_op_parallelism_threads=1)
  sess = tf.Session(graph=tf.get_default_graph(), config=session_conf)
  K.set_session(sess)
```

**Comment by 14H034160212** [comment]
I solve that issue when I updated the code in the `util.py`.
```
    seed_value = 0
    os.environ['PYTHONHASHSEED'] = str(seed_value)
    random.seed(seed_value)
    np.random.seed(seed_value)
```
